src/utilities/gql.ts

```diff
@@ -1184,6 +1218,7 @@ const gql = {
         metadata
         seq_id
         simulation_dataset_id
+        updated_at
```

Comment:
I'm seeing a subscribe error due to this line, which is causing the `filteredExpansionSequences` not to display.

```
"field 'updated_at' not found in type: 'sequence'"
```

Reply:
Ah yep, think I mixed up sequence and user sequence fields, will fix.

Reply:
Yeah not sure why some of these were removed. For now let's just support what is defined on develop for the `sequence` table: https://github.com/NASA-AMMOS/aerie/blob/develop/sequencing-server/sql/sequencing/tables/sequence.sql

Reply:
Fixed